hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/TestLocalMetadataStore.java

```diff
@@ -169,7 +171,9 @@ private static void populateMap(Cache<Path, LocalMetadataEntry> cache,
   private static void populateEntry(Cache<Path, LocalMetadataEntry> cache,
       Path path) {
     FileStatus fileStatus = new FileStatus(0, true, 0, 0, 0, path);
-    cache.put(path, new LocalMetadataEntry(new PathMetadata(fileStatus)));
+    S3AFileStatus s3aStatus = S3AFileStatus.fromFileStatus(
```

Comment:
There's *a lot* of calls like this. Cleaner to just create S3AFileStatus directly in most of these cases, yeah?

Reply:
Good observation.  I'll clean that up.

Reply:
Addressed in https://github.com/apache/hadoop/pull/675/commits/3488b20da09bc6ceb8bcfca140aefdbc8ea6cca4